src/job.rs

```diff
@@ -41,11 +41,27 @@ enum Thread {
     Smtp = 5000,
 }
 
-#[derive(Debug, Display, Copy, Clone, PartialEq, Eq, FromPrimitive, ToPrimitive)]
-enum TryAgain {
-    Dont,
-    AtOnce,
-    StandardDelay,
+/// Job try result.
+#[derive(Debug, Display)]
+pub enum Try {
```

Comment:
`JobStatus` would be more discriptive?  E.g. in configure/mod.rs above it just imports `crate::job::*` (which is possibly a bad pattern itself) and then a lonely `Try` appears.  I guess this would be better as `job::Try`, but even better as `job::Status`, `JobStatus` seems a compromise I could live with ;).

Reply:
`clippy` has some disabled-by-default checks against names that contain the name of the module, I guess it is not a good style. So it should be job::Status, right.

Reply:
Fixed it by renaming into `job::Status` and importing `job` without `*`.